pkgs/scotch/Make.inc.linux

```diff
@@ -9,9 +9,9 @@ CAT		= cat
 CCS		= $(MPICC)
 CCP		= $(MPICC)
 CCD		= $(MPICC)
-CFLAGS		= -fPIC -O3 -DCOMMON_FILE_COMPRESS_GZ -DCOMMON_PTHREAD -DCOMMON_RANDOM_FIXED_SEED -DSCOTCH_RENAME -Drestrict=__restrict
+CFLAGS		= -I$(ZLIB_DIR)/include -fPIC -O3 -DCOMMON_FILE_COMPRESS_GZ -DCOMMON_PTHREAD -DCOMMON_RANDOM_FIXED_SEED -DSCOTCH_RENAME -Drestrict=__restrict
 CLIBFLAGS	=
-LDFLAGS		= -lz -lm -lrt
+LDFLAGS		= -L$(ZLIB_DIR)/lib -lz -lm -lrt
```

Comment:
Ah, it doesn't matter at all for a static library, I didn't realize you were building a static library.

+1 to merge then.